src/components/pages/repository/RepoTable.tsx

```diff
@@ -0,0 +1,281 @@
+        &.-sort-asc {
+          box-shadow: inset 0 3px 0 0 ${theme.colors.secondary};
+        }
+        &.-sort-desc {
+          box-shadow: inset 0 -3px 0 0 ${theme.colors.secondary};
+        }
+        &:focus {
+          outline: none;
+        }
+      }
+    }
+    & .rt-thead .rt-th {
+      ${theme.typography.data};
+      font-weight: bold;
+      text-align: left;
+      color: ${theme.colors.accent_dark};
+    }
+    & .rt-td {
+      ${theme.typography.data};
+      & div {
+        text-align: left !important;
+      }
+    }
+    & .rt-td .td-actions {
+      width: 100%;
+      display: inline-block;
+      text-align: center;
+    }
+    & .rt-tr-group {
+      border-bottom: none;
+      border-top: none;
+      &:hover {
+        background: ${theme.colors.grey_highlight};
+      }
+    }
+    & .rt-tr-group .rt-tr.-even {
+      &:hover {
+        background: ${theme.colors.grey_highlight};
+      }
+    }
+    & .rt-tr-group .rt-tr.-odd {
+      background-color: ${theme.colors.grey_1};
+      &:hover {
+        background: ${theme.colors.grey_highlight};
+      }
+    }
+    & .pagination-bottom {
+      & .-pagination {
+        padding: 0px;
+        height: 45px;
+        box-shadow: none;
+        border: none;
+        ${theme.typography.label};
+        font-weight: normal;
+        & .-pageJump {
+          border: none;
+          display: flex;
+          font-size: 13px;
+          justify-content: space-around;
+          & .-pagination_button {
+            cursor: pointer;
+            background-position: center;
+            background-color: ${theme.colors.white};
+            margin: 0 6px;
+            height: 24px;
+            width: 24px;
+            border-radius: 25px;
+            text-align: center;
+            padding-top: 3px;
+          }
+          & .-pagination_button.-current {
+            background-color: ${theme.colors.secondary_1};
+          }
+          & .-toStart,
+          & .-previous,
+          & .-next,
+          & .-toEnd {
+            font-weight: bold;
+          }
+          & .-toStart,
+          & .-toEnd {
+            letter-spacing: -2px;
+          }
+        }
+      }
+      & select {
+        padding: 5px 10px 5px 5px;
+        appearance: none;
+        width: 45px;
+        text-align: left;
+        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='10' height='10' viewBox='0 0 12 12'%3E%3Cpath fill='%23151c3d' fill-rule='evenodd' d='M9.952 3.342c.468-.456 1.228-.456 1.697 0 .234.228.351.526.351.825 0 .298-.117.597-.351.825l-4.8 4.666c-.469.456-1.23.456-1.697 0l-4.8-4.666c-.47-.456-.47-1.194 0-1.65.468-.456 1.228-.456 1.696 0L6 7.184l3.952-3.842z'/%3E%3C/svg%3E");
+        background-repeat: no-repeat;
+        background-position: bottom 7px right 4px;
+      }
+    }
+  }
+`;
+
+const RepoTable = (props: PageContentProps) => {
+  const theme = useTheme();
+
+  return (
+    <div css={getTableStyle(theme)}>
```

Comment:
I believe `css` calls will get the `theme` as an arg if you give it a function.
eg. <div css={theme => getTableStyle(theme)}/>
which means you could avoid the `useTheme`